src/System.Diagnostics.EventLog/tests/Helpers.cs

```diff
@@ -18,6 +18,17 @@ internal class Helpers
         public static bool IsElevatedAndSupportsEventLogs { get => AdminHelpers.IsProcessElevated() && SupportsEventLogs; }
         public static bool SupportsEventLogs { get => PlatformDetection.IsNotWindowsNanoServer && PlatformDetection.IsNotWindowsIoTCore; }
 
+        // Retry that eats exceptions: for "best effort cleanup"
+        public static void RetrySilently(Action func)
+        {
+            try
+            {
+                Retry(func);
+            }
+            catch
+            {}
+        }
```

Comment:
Seems like this is only ever used for EventLog.Delete, and using it for something else would potentially be dangerous.  So can it just be:
```C#
public static void DeleteNoFailures(EventLog log)
{
    try { EventLog.Delete(log); }
    catch { }
}
```
or something like that?